Tribler/Main/Utility/GuiDBHandler.py

```diff
@@ -73,6 +75,10 @@ def Add(self, sender, workerFn, args=(), kwargs={}, name=None, delay=0.0, uId=No
         """The sender will send the return value of
         workerFn(*args, **kwargs) to the main thread.
         """
+        if self.quitting:
+            self._logger.debug("GUIDBHandler: rejecting task(%s), quitting flag is set.", name)
```

Comment:
I would make this a warning to not catch developers with the pants down it they expect stuff to still be called when shutting down.